src/MetaDataAPI/Response/ValidationErrorResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Net;
+using FluentValidation.Results;
+
+namespace MetaDataAPI.Response;
+
+public class ValidationErrorResponse : LambdaResponse
+{
+    public ValidationErrorResponse(ValidationResult validationResult)
+        : base(
+            body: validationResult.ToString($"{Environment.NewLine}"),
```

Comment:
is it mistype?

Reply:
If you are about "ToString()"

This is a special method of "FluentValidation" library in "ValidationResult" object
![image](https://github.com/The-Poolz/LockDeal-NFT-API/assets/66203238/9a77035f-58ae-4d15-ac64-7dce3126cc6b)